index.html

```diff
@@ -89,9 +116,10 @@
     templating and pre-rendering of content likely to become visible in the
     near future. These tasks must share the event loop with other time-critical
     operations, such as reacting to input and performing script-based
-    animations using `requestAnimationFrame` ([[ANIMATION-TIMING]]). These
+    animations using <code><a>requestAnimationFrame</a></code>. These
```

Comment:
If you need `<code>` anywhere for a method or attribute, it means that the method or attribute is not correctly defined. 

To correctly define an attribute or method from another spec, you do it like this:

```HTML
<code><dfn data-cite="html#dom-window-requestanimationframe">requestAnimationFrame()</dfn></code> is defined in [[!html]].
```